.env.template

```diff
@@ -52,7 +52,7 @@ POSTGRES_CONNECTION_STRING=
 # Managed Redis (default: false)
 REDIS_MANAGED=false
 # Managed Grafana (default: false), read more in ./terraform/grafana/README.md
-MANAGED_GRAFANA=false
+GRAFANA_MANAGED=false
```

Comment:
The variable name was incorrect, see /Makefile line 53 and /variables.tf line 346